### PR TITLE
added virutal_network_to_fabric_site

### PR DESCRIPTION
**Pull request by kuba-mazurkiewicz**
#43

### CODE REVIEW

File: gen/definitions/virtual_network_to_fabric_site.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+name: Virtual Network To Fabric Site
+rest_endpoint: /dna/intent/api/v1/business/sda/virtual-network
+no_update: true
+no_import: true
+no_data_source: true
+skip_minimum_test: true
+res_description: Add virtual network (VN) to SDA Fabric Site
```
Comment (jabielecki) [inline gen/definitions/virtual_network_to_fabric_site.yaml:8]: It seems to be able to delete as well as add so maybe something like:

```suggestion
res_description: Manage assignment of a virtual network (VN) to an SD-Access Fabric Site.
```

File: gen/definitions/virtual_network_to_fabric_site.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+doc_category: SDA
+attributes:
+  - model_name: virtualNetworkName
+    type: String
+    mandatory: true
+    data_source_query: true
+    description: Virtual Network Name, that is created at Global level
```
Comment (jabielecki) [inline gen/definitions/virtual_network_to_fabric_site.yaml:15]: This leaves me wondering. Let's say I create a VN using code below, but I have no idea whether it is "at Global level" or how to verify that:

```
  resource "catalystcenter_fabric_virtual_network" "test" {
    virtual_network_name = "SDA_VN1"
    is_guest             = false
    sg_names             = ["Employees"]
  }
```

File: gen/definitions/virtual_network_to_fabric_site.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+name: Virtual Network To Fabric Site
```
Comment (jabielecki) [inline gen/definitions/virtual_network_to_fabric_site.yaml:2]: Nice name. 👍

File: gen/definitions/virtual_network_to_fabric_site.yaml (hunk 2 above)
Comment (kuba-mazurkiewicz) [reply gen/definitions/virtual_network_to_fabric_site.yaml:15]: when you create VN with this code, you should be able to verify it here: 
/dna/provision/sd-access/virtualNetworks/list/layer_3/global

File: gen/definitions/virtual_network_to_fabric_site.yaml (hunk 3 above)
Comment (kuba-mazurkiewicz) [reply gen/definitions/virtual_network_to_fabric_site.yaml:2]: in API its not intuitive. You have: 
- Add virtual network with scalable groups (/dna/intent/api/v1/virtual-network)
- Add VN in fabric (/dna/intent/api/v1/business/sda/virtual-network)

Second resource is used to add VN (created using first resource) to fabric site.

File: gen/definitions/virtual_network_to_fabric_site.yaml (hunk 1 above)
Comment (kuba-mazurkiewicz) [reply gen/definitions/virtual_network_to_fabric_site.yaml]: sounds good, changed that

File: gen/definitions/virtual_network_to_fabric_site.yaml (hunk 2 above)
Comment (jabielecki) [reply gen/definitions/virtual_network_to_fabric_site.yaml:15]: No no, I don't mean I need explanation here. I was trying to say that `description` needs to be made more clear.

**Review by danischm — APPROVED** [review]

**Pull request merged**